integr8ly/installation
# INTLY-934 Add OpenShift monitoring federation

Federate with the Prometheus in the OpenShift Monitoring from the
middleware monitoring namespace instead of using our own instance
of Kube State Metrics.

Verification:
- Run the installer and ensure it completes
- Ensure Kube State Metrics is not in middleware-metrics
- Ensure Alerts are passing (they may be pending for a while)
- Run the uninstaller and ensure it completes
